.tekton/insights-chrome-pull-request.yaml

```diff
@@ -670,6 +733,16 @@ spec:
       optional: true
   taskRunTemplate:
     serviceAccountName: build-pipeline-insights-chrome
+    podTemplate:
+      hostAliases:
+      - ip: "127.0.0.1"
```

Comment:
I'm unsure how the network stack is set up inside the container/pod. I've occasionally bumped into the issue where locahost doesn't work but the internal adapter address (0.0.0.0) does.